core/sector_storage/impl/manager_impl.cpp

```diff
@@ -778,6 +785,208 @@ namespace fc::sector_storage {
     ;
   }
 
+  void ManagerImpl::serveHTTP(const http::request<http::dynamic_body> &request,
+                              http::response<http::dynamic_body> &response) {
+    response.version(request.version());
+    response.keep_alive(false);
+
+    switch (request.method()) {
+      case http::verb::get:
+        createResponseGet(request, response);
+        break;
+      case http::verb::delete_:
+        createResponseDelete(request, response);
+        break;
+      default:
+        response.result(http::status::method_not_allowed);
+        break;
+    }
+  }
+
+  void ManagerImpl::createResponseGet(
+      const http::request<http::dynamic_body> &request,
+      http::response<http::dynamic_body> &response) {
+    std::regex stat_rgx(R"(\/remote\/stat\/([\w-]+))");
+    std::regex get_sector_rgx(R"(\/remote\/([\w]+)\/([\w-]+))");
```

Comment:
(should we use std::regex or boost::regex?)
Can these two regexes be simplified to split by '/' and equality checks ([0] == "remote", [1] == "stat")?

Reply:
Checking for equality and immediately pull out the parameters is easier this way